modules/ROOT/pages/index.adoc

```diff
@@ -48,6 +48,7 @@ Both the public and private content in Exchange can consist of the following kin
 * REST APIs - A RAML file or OAS file that specifies an API. These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
 * SOAP APIs - A WSDL file that specifies an API.
 * HTTP APIs - A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager.
+* API Groups - A set of related APIs that can be published, updated, and accessed by developers' client applications as a single unit.
```

Comment:
Reuse from the intro text:
Using API Groups enables you to bundle your APIs and resources to solve specific user needs. Instead of using individual resources and APIs from a list, the users can request access to and use these in a package that solves a specific problem for them.

Reply:
Done. Thank you!